src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Styles/ComboBox.xaml

```diff
@@ -208,20 +209,27 @@
                                             IsHitTestVisible="False"
                                             TextElement.Foreground="{TemplateBinding Foreground}" />
                                     </Grid>
-                                    <Grid Grid.Column="1" Margin="{StaticResource ComboBoxChevronMargin}">
-                                        <TextBlock
-                                            x:Name="ChevronIcon"
-                                            Margin="0"
-                                            VerticalAlignment="Center"
-                                            FontSize="{StaticResource ComboBoxChevronSize}"
-                                            Foreground="{DynamicResource ComboBoxDropDownGlyphForeground}"
-                                            RenderTransformOrigin="0.5, 0.5"
-                                            FontFamily="{DynamicResource SegoeFluentIcons}"
-                                            Text ="{StaticResource ComboBoxChevronDownGlyph}">
-                                            <TextBlock.RenderTransform>
-                                                <RotateTransform Angle="0" />
-                                            </TextBlock.RenderTransform>
-                                        </TextBlock>
+                                    <Grid Grid.Column="1" 
+                                        VerticalAlignment="Center" 
+                                        HorizontalAlignment="Center"
+                                        Margin="{StaticResource ComboBoxChevronMargin}">
+                                        <Border 
```

Comment:
Why not use a button with textblock, I think that will be sufficient and we won't need to add any new resources.

Reply:
Button styles are not going to align with chevron icon style. Even my first approach was to achieve this was through button.